index.html

```diff
@@ -9,72 +9,75 @@
         <link rel="preconnect" href="https://fonts.googleapis.com">
         <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
         <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@400;500;700&family=Poppins:wght@400;700&family=Roboto:wght@400;700&display=swap" rel="stylesheet">
+        <script defer src="index.js"></script>
     </head>
     <body>
-        <header class="nav">
-            <a href="#" class="nav-title">Innocent N.</a>
-            <nav class="nav-nav">
-                <img src="./images/menu.png" alt="menu-icon" class ="nav-menu" />
-                <ul class="links">
-                    <li>
-                        <a href="#">Portfolio</a>
-                    </li>
-                    <li>
-                        <a href="#">About</a>
-                    </li>
-                    <li>
-                        <a href="#">Contact</a>
-                    </li>
-                    <li>
-                        <a href="#">
-                            <img src="./images/Icon - Mail.svg" alt="Mail-icon" class="mail-icon" />
```

Comment:
- [ ] You did a great job 😃 at adding the functionality of the mobile menu, but kindly try to stick to the Figma Design to make your mobile menu like the Figma as I see here that the color of the links in the mobile menu is different from the Figma, so please try to put the correct color to the links to be like the Figma 💯
- [ ] Also it will be better if you decrease the margin-top of the icon ```x``` in the mobile menu as you see it is different from the Figma Design 

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/224971302-3e3a39ed-b6fb-4adb-8dbe-ea615584097b.png)   |   ![image](https://user-images.githubusercontent.com/83360636/224971566-887c2920-6655-4721-8466-267e93563df7.png) |